### PR TITLE
Added path trick to the picklist example

### PR DESCRIPTION
**Pull request by SergeoLacruz**
Because the team complained about the /  :-)

### CODE REVIEW

**Comment by SergeoLacruz** [comment]
Hi,
Thanks for the honor. Of course I am willing to do that. 
I will not be able to contribute much code because that is not my  profession. But I have experience with HW design methods, 
tools and processes for all the infrastructure around the schematic. See you,

Michael

File: docs/report/pack.md [hunk 1]
```diff
@@ -52,3 +52,20 @@ that is needed for the build. This will result in the following printout:
 
 {% with id="report-options", url="report/picklist.png", description="Picklist Example" %} {% include "img.html" %} {% endwith %}
 
+For those of you who would like to replace the "/" by something else because it is hard 
+the read in certain fonts use the following trick:
```
Comment (SchrodingersGat) [inline docs/report/pack.md:56]: @SergeoLacruz not sure what you mean here about "fonts"?

Comment (matmair) [reply docs/report/pack.md:56]: @SchrodingersGat probably font as in font face/font family.

**Comment by SchrodingersGat** [comment]
Thanks, makes sense :)

**Pull request merged**